.github/workflows/build.yaml

```diff
@@ -21,14 +21,9 @@ jobs:
           distribution: 'zulu'
           java-version: 19
 
-      # For cinterop issues. Remove with Kotlin 1.8.
-      - uses: maxim-lobanov/setup-xcode@v1
-        with:
-          xcode-version: '13.4.1'
-
       - uses: gradle/gradle-build-action@v2
 
-      - run: ./gradlew build dokkaHtmlMultiModule --parallel -PredwoodNoSamples
+      - run: ./gradlew kotlinUpgradeYarnLock build dokkaHtmlMultiModule --parallel -PredwoodNoSamples
```

Comment:
It seems like as of 1.8 `yarn.lock` is checked on each run so I've committed the correct yarn file for building with samples and bypassed the check on CI by always running `kotlinUpgradeYarnLock` first.